demo-v5/package.json

```diff
@@ -34,18 +34,16 @@
     "@netlify/ipx": "latest",
     "co-body": "^6.1.0",
     "cookie": "^0.6.0",
-    "download": "^8.0.0",
     "form-data": "^4.0.0",
     "fs-extra": "^10.0.0",
     "gatsby-plugin-netlify": "^5.1.0",
     "jest": "^29.0.0",
     "linkfs": "^2.1.0",
-    "multer": "^1.4.2",
+    "multer": "^1.4.5-lts.1",
```

Comment:
Any reason we want the `lts.1` version?

Reply:
This is only not deprecated version from v1 line and v2 don't have stable version released. `latest` also point to that version:
```
> npm info multer@latest version
1.4.5-lts.1
```

It's a bit weird, but 🤷

Reply:
Merge away then! 😄